gum/backend-arm64/gumstalker-arm64.c

```diff
@@ -2673,6 +2691,36 @@ gum_exec_block_write_call_invoke_code (GumExecBlock * block,
     gum_arm64_writer_put_ldr_reg_value (cw, ic_load_real_address_ref,
         GUM_ADDRESS (ret_real_address));
   }
+
+  if (target->reg != ARM64_REG_INVALID)
+  {
+    gum_arm64_writer_put_label (cw, keep_this_blr);
+
+    gum_exec_ctx_write_epilog (block->ctx, GUM_PROLOG_MINIMAL, cw);
+    gum_arm64_writer_put_blr_reg (cw, target->reg);
+
+    gum_exec_block_write_jmp_transfer_code (block, &next_insn_as_target,
+        GUM_ENTRYGATE (jmp_over_call_reg), gc);
+  }
+}
+
+static GumAddress
```

Comment:
- The `is_` part of this name makes it sound like it's returning a `gboolean` – so I suppose that would be clearer?
- This is pretty high-level, so perhaps it would be good to move this right after `gum_stalker_exclude`? (Which also means we don't have to put a forward-declaration.)